comp/metadata/inventoryhost/inventoryhostimpl/inventoryhost.go

```diff
@@ -230,6 +236,10 @@ func (ih *invHost) fillData() {
 	ih.data.CloudProviderSource = cloudproviders.GetSource(cloudProvider)
 	ih.data.CloudProviderHostID = cloudproviders.GetHostID(context.Background(), cloudProvider)
 	ih.data.OsVersion = osVersionGet()
+
+	gpgcheck, repoGPGCheck := pkgSigningGet()
```

Comment:
you mean the public method should not be part of the component itself, and be set in a separate package "under" the component?

Reply:
We try to separate helper/utils function from component implementation. In this case, we only need to import the helpers so having a dedicated package for this would be nice. This guarantee that the component doesn't offers any API outside FX and that the helpers is not tight to any state from the component.